rental.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+require_relative './book'
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(book, person, date)
+    @date = date
+    @person = person
+    @book = book
+    person.rentals << @book
+    book.rentals << @person
```

Comment:
- [OPTIONAL] The rental is adding itself to the @book achieving the 'belongs-to' association, like `book.rentals << self` and rental is adding itself to the @person achieving the 'belongs-to' association, like `person.rentals << self`. Please kindly fix that.

Reply:
Hey if this won't bother, @Mihndim2020. From what I read if I did something like "person.rentals << @book" that would send the name of the book only, to the person, If I send self instead that would be sending the whole instance of rental correct ? and from what you said and from what I understood better from the requirement at this point it's more beneficial for later use to have the whole instance of the rental stored in book/person.rental rather than the person's name/ book's name